v2/my-garden/src/app/writings/[id]/page.tsx

```
@@ -25,11 +25,14 @@ export default async function Page({ params }: WritingProps) {
             <div className='min-w-full md:container px-8  md:px-28 lg:px-60'>
 
                 {/* Header */}
-                <section className="md:leading-10 leading-8 border-b-2 border-gray-400 border-opacity-40">
+                <section className="md:leading-10 leading-8 pb-4 border-b-2 border-gray-400 border-opacity-40">
                     <p className="text-3xl md:text-4xl py-4">{writing.title}</p>
-                    <section className="text-slate-200 text-md pb-4">
+                    <section className="text-slate-200 text-md">
                         <p>{writing.description}</p>
                     </section>
+                    <div className="flex justify-start py-2">
+                        <LikeButton id={writing.id} type="post" />
```

ESLint

[error] 'LikeButton' is not defined.

(react/jsx-no-undef)